src/index.test.ts

```
@@ -0,0 +1,35 @@
+const originalConsoleError = console.error;
+let consoleOutput: string[] = [];
+
+describe('Main application', () => {
+  beforeEach(() => {
+    // Setup console mock
+    consoleOutput = [];
+    console.log = jest.fn((...args) => {
+      consoleOutput.push(args.join(' '));
+    });
+    console.error = jest.fn((...args) => {
+      consoleOutput.push('ERROR: ' + args.join(' '));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)